packages/tracing/src/transaction.ts

```diff
@@ -42,7 +42,11 @@ export class Transaction extends SpanClass implements TransactionInterface {
 
     this._name = transactionContext.name || '';
 
-    this.metadata = transactionContext.metadata || {};
+    this.metadata = {
+      ...transactionContext.metadata,
+      spanMetadata: {},
```

Comment:
Shouldn’t the order here be flipped so that we don’t always override `spanMetadata`?

Reply:
> Note that there should never be incoming span metadata because the only span at that point is the transaction itself, and its metadata can and does already live in the main metadata object. This means there's no need to do `span: metadata.span || {}` when initializing it. 

^^ I think this covers it, no?

Reply:
Ah yes - that makes sense.